diagnostic.md

```diff
@@ -7,6 +7,7 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+    the router keeps tracks of the urls or the path of each templates.  Inside each route, you will want to include a model hook, which pass the information into the template.  it helps load the appropriate template and the appropriate model.
```

Comment:
Yes, the Ember Application Router defines the route, and the Ember Route specifies how to load model data using the model hook.